pandas/tests/io/json/test_json_table_schema.py

```diff
@@ -451,12 +451,17 @@ def test_to_json_categorical_index(self):
         assert result == expected
 
     def test_date_format_raises(self, df_table):
-        msg = (
+        error_msg = (
```

Comment:
This is kind of strange to both warn and raise. I guess there is no easy way to detangle this for the table orient?

Reply:
I agree it doesn't look pretty, but this is the best I could come up with, any help would appreciated